### ISSUE TITLE
Lexmark Printer setup

### ISSUE THREAD

**Issue by stephanstan** [stephanstan/stephanstan.github.io#33]
## TL;DR
local url - seems to occasionally increment like
from 
http://192.168.1.69/#/Settings/Email
to current
http://192.168.1.70/#/Settings/Email

sometimes the smtp portnumber changes -
587 used to work
465 also works

key to resolution
https://www.youtube.com/watch?v=qV77IuQL75c

Gmail accounts -> Security -> 

### Article Link
[local Lexmark Printer setup url](http://192.168.1.70/#/Settings/Email)
## Key Takeaways
* use telnet to check if gmail smtp is accessible - https://smallbusiness.chron.com/connect-gmail-telnet-26966.html

![image](https://user-images.githubusercontent.com/543700/92334214-3161e700-f049-11ea-95d1-ba527234dccf.png)

* 

## Useful Code Snippets
```python
#Add your codes here


```
![image](https://user-images.githubusercontent.com/543700/92334184-ed6ee200-f048-11ea-9ea4-e3d2b6e9a9c8.png)

## Useful Tools
* 
*
